gnomad_qc/v4/resources/release.py

```diff
@@ -19,6 +19,46 @@
     RELEASES,
 )
 
+FREQUENCY_README = """
+The 'freq' row annotation contains allele frequency information aggregated by certain
+sample groupings within the dataset within an array. Each element of the array contains
+a grouping's alternate allele count (AC), alternate allele frequency (AF), total number
+ofalleles (AN), and the count of homozygous alternate individuals (homozygote_count).
```

Comment:
```suggestion
The 'freq' row annotation is an array that contains allele frequency information. Each element of the array is a struct that contains the alternate allele count (AC), alternate allele frequency (AF), total number of alleles (AN), and number of homozygous alternate individuals (homozygote_count) for a specific sample grouping.
```
now I'm wondering if I should reword the help/faq as well